libibverbs/verbs.h

```diff
@@ -1290,6 +1295,9 @@ struct ibv_qp_ex {
 			     uint64_t remote_addr);
 	void (*wr_rdma_write)(struct ibv_qp_ex *qp, uint32_t rkey,
 			      uint64_t remote_addr);
+	void (*wr_rdma_flush)(struct ibv_qp_ex *qp, uint32_t rkey,
```

Comment:
This is ABI it ahas to go at the end of the struct

Reply:
Got it. updated